backend/ipmi.pm

```diff
@@ -97,6 +97,11 @@ sub restart_host {
 sub do_start_vm {
     my ($self) = @_;
 
+    # reset ipmi main board if switch on
+    if ($bmwqemu::vars{IPMI_BACKEND_MC_RESET} eq 'yes') {
```

Comment:
So we set this in the workers.ini for the less stable machines?

Reply:
yes, it is supposed to set in workers.ini, and better for all ipmi machines(all ipmi machines are unstable under long time working).

Reply:
So why have a variable at all? For developing?

Reply:
In any case: Add comments here and in the docs/ on when and why to use this variable with what content.

Reply:
Thanks for your suggestion. I will add them.

Why having the variable is mainly that this solution to improve stability via mc reset needs to be proved by large practical openqa jobs statistics. We *may* need to switch on/off on some ipmi machines according to real run results and maybe different behaviors on different machines. So having this option provided can easily tune.